src/components/Page/LandingPageWidgets.vue

```diff
@@ -1,6 +1,16 @@
 <template>
 	<div class="landing-page-widgets">
-		<MembersWidget v-if="!isPublic" />
+		<div class="first-row-widgets">
+			<MembersWidget v-if="!isPublic" :current-collective="currentCollective" />
+			<NcButton v-if="'contacts' in OC.appswebroots" :href="teamUrl">
+				<template #icon>
+					<TeamsIcon :size="20" />
+				</template>
+				<template #default>
+					{{ t('collectives','Team') }}
```

Comment:
I would call it "Team overview". What do others think @juliushaertl @nextcloud/designers?

Reply:
Team overview sounds good to me 👍